specs/protocol/configurability.md

```diff
@@ -0,0 +1,80 @@
+# OP Stack Configurability
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Consensus Parameters](#consensus-parameters)
+- [Admin Roles](#admin-roles)
+- [Service Roles](#service-roles)
+- [Policy Parameters](#policy-parameters)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+When deploying the OP Stack software to a production environment,
+certain parameters about the protocol can be configured. These
+configurations can include a variety of parameters which affect the
+resulting properties of the blockspace in question.
+
+There are four categories of OP Stack configuration options:
+
+- **Consensus Parameters**: Parameters and properties of the chain that may
+  be set at genesis and fixed for the lifetime of the chain, or may be
+  changeable through a privileged account.
+- **Policy Parameters**: Parameters of the chain that can be changed without breaking consensus. Consensus is enforced while policy includes the set of possible behaviors within consensus. Policy is ultimately constrained by consensus. Changing policy paramters won't impact how an OP chain is classified; a chain can still be considered standard if it has it's own custom policy parameters.
```

Comment:
```suggestion
- **Policy Parameters**: Parameters of the chain that might be changed without breaking consensus. Consensus is enforced by the protocol, while policy parameters may be changeable within constraints imposed by consensus.
```

Reply:
Same kinda note as other language suggeston.  I think the original wording is right most of the time, but strictly speaking, it is possible to impose social restrictions on acceptable policy that are more strict than consensus/protocol.